test/e2e/flowaggregator_test.go

```diff
@@ -1393,6 +1398,15 @@ func checkEgressInfoClickHouse(t *testing.T, record *ClickHouseFullRow, egressNa
 	assert.Equal(t, egressIP, record.EgressIP, "Record does not have correct egressIP")
 }
 
+func checkL7FlowExporterData(t *testing.T, record, appProtocolName string) {
+	assert.Containsf(t, record, fmt.Sprintf("appProtocolName: %s", appProtocolName), "Record does not have correct Layer 7 protocol Name")
+}
+
+func checkL7FlowExporterDataClickHouse(t *testing.T, record *ClickHouseFullRow, appProtocolName string) {
+	assert.Equal(t, record.AppProtocolName, appProtocolName, "Record does not have correct Layer 7 protocol Name")
+	assert.NotEmpty(t, record.HttpVals, "Record does not have httpVals")
+}
```

Comment:
Do we need these two functions? I think they are only used once and the value of `appProtocolName` is always `http`?

Reply:
the two functions are for two different records, one for click house other one for collector

Reply:
Yes. But each one of them is used only once. I was thinking maybe it's better to add line#1402 to line#1927 and line#1406-1407 to line#1939

Reply:
Sure , we can do that

Reply:
let's keep this as it is, this PR has been pending for a while